### PR TITLE
Updated Translation model to gpt4o and also updated system prompt

### LINKED ISSUE #45: openai weird response
cleaned_string='There is no exact translation for this series of letters as it does not form a recognizable word or phrase in English.'

probably need to change our prompt to openai to never give this or a very specific string to handle

### PR DESCRIPTION
**Pull request by ShaneKoNaung**

### CODE REVIEW

File: error_analysis_gpto.ipynb
Comment (mattkanwisher) [inline error_analysis_gpto.ipynb:1]: move notebooks to their own folder, the root folder is way to busy

File: get_openai_responses.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+dialogue_box_folder = Path('../data/dialogue_boxes_en_gpt4o/')
+
+# Create the dialogue box folder if it doesn't exist
+dialogue_box_folder.mkdir(parents=True, exist_ok=True)
+
+ocr_responses = {}
+
+# Iterate over all image files in the specified folder
+for image_path in images_folder.glob('*.jpg'):
+    try:
```
Comment (mattkanwisher) [inline get_openai_responses.py]: can we make like an openai package or something so we don't have new files in root

File: grab_ss.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+from Xlib import display
+import numpy as np
+from PIL import Image, ImageGrab
+import subprocess
+import time
+
+def find_window_id(window_name):
+    root = display.Display().screen().root
+    window_id = None
+
```
Comment (mattkanwisher) [inline grab_ss.py]: why I this screenshot grabbing different then the other files we have?

File: get_openai_responses.py (hunk 1 above)
Comment (ShaneKoNaung) [reply get_openai_responses.py]: These scripts are for generating some test samples. They are not really needed anymore. I'll keep them in a scripts folder.

File: grab_ss.py (hunk 2 above)
Comment (ShaneKoNaung) [reply grab_ss.py]: This actually is my fault. Screen grabbing wasn't working well on this laptop, So, I was trying different methods. This shouldn't have been committed.

File: error_analysis_gpto.ipynb
Comment (ShaneKoNaung) [reply error_analysis_gpto.ipynb:1]: Done.

**Pull request merged**